engine/src/main/java/org/terasology/logic/actions/PlaySoundActionComponent.java

```diff
@@ -34,6 +34,10 @@
     public PlaySoundActionComponent() {
     }
 
+    /**
+     * Add sounds to list of sounds
```

Comment:
This again could be written with a bit more meaning, and the `sounds` param needs a line as well.

In this case `sounds` is actually a "vararg" parameter, meaning you can pass any number of StaticSound variables to this method. Which is in fact a constructor, not just a regular method. So you could say something like "Creates a new instance of `PlaySoundActionComponent` adding all provided sounds via vararg to the instance variable `sounds`" and the `@param sounds` could say "the provided sounds via vararg"